blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -313,4 +323,7 @@ public void run() {
         startEventListenerModules();
         log.info("All blueflood services started");
     }
+
+    @VisibleForTesting
+    public static boolean SkipInstantiateRestartGauge = false;
```

Comment:
Can you make the variable start with smaller case? Usually for booleans, it is preceded with "is" or something like that but thats upto you.